app/screens/onboarding/slide.tsx

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {View, useWindowDimensions} from 'react-native';
+import Animated, {Extrapolate, interpolate, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+
+import {OnboardingItem} from '@typings/screens/onboarding';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    item: OnboardingItem;
+    theme: Theme;
+    scrollX: Animated.SharedValue<number>;
+    index: number;
+    lastSlideIndex: number;
+};
+
+export const ONBOARDING_CONTENT_MAX_WIDTH = 520;
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    image: {
+        justifyContent: 'center',
+        maxHeight: 180,
+    },
+    title: {
+        color: theme.centerChannelColor,
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        maxWidth: ONBOARDING_CONTENT_MAX_WIDTH,
+        marginBottom: 12,
+    },
+    fontTitle: {
+        marginTop: 32,
+        ...typography('Heading', 1000, 'SemiBold'),
+    },
+    fontFirstTitle: {
+        ...typography('Heading', 1200, 'SemiBold'),
+        paddingTop: 48,
+        letterSpacing: -1,
+    },
+    widthLastSlide: {
+        paddingHorizontal: 50,
+    },
+    firstSlideInitialPosition: {
+        left: 200,
+        opacity: 0,
+    },
+    description: {
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        color: changeOpacity(theme.centerChannelColor, 0.64),
+        ...typography('Body', 200, 'Regular'),
+        maxWidth: ONBOARDING_CONTENT_MAX_WIDTH,
+    },
+    itemContainer: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+}));
+
+const FIRST_SLIDE = 0;
+
+const SlideItem = ({theme, item, scrollX, index, lastSlideIndex}: Props) => {
+    const {width} = useWindowDimensions();
+    const styles = getStyleSheet(theme);
+
+    /**
+     * Code used to animate the first image load
+     */
+    const [firstLoad, setFirstLoad] = useState(true);
+
+    const initialImagePosition = useSharedValue(width);
+    const initialTitlePosition = useSharedValue(width);
+    const initialDescriptionPosition = useSharedValue(width);
```

Comment:
Same worries regarding width when changing from portrait to landscape.

Reply:
the dimensions change work fine in the simulator.